cypress/e2e/data-import/importing-marc-bib-files/import-edit-derive-bib-with-minimum-fields.cy.js

```diff
@@ -0,0 +1,93 @@
+import getRandomPostfix from '../../../support/utils/stringTools';
+import TestTypes from '../../../support/dictionary/testTypes';
+import DevTeams from '../../../support/dictionary/devTeams';
+import Permissions from '../../../support/dictionary/permissions';
+import TopMenu from '../../../support/fragments/topMenu';
+import DataImport from '../../../support/fragments/data_import/dataImport';
+import Users from '../../../support/fragments/users/users';
+import JobProfiles from '../../../support/fragments/data_import/job_profiles/jobProfiles';
+import Logs from '../../../support/fragments/data_import/logs/logs';
+import InventoryInstance from '../../../support/fragments/inventory/inventoryInstance';
+import QuickMarcEditor from '../../../support/fragments/quickMarcEditor';
+import { JOB_STATUS_NAMES } from '../../../support/constants';
+import inventorySteps from '../../../support/fragments/inventory/inventorySteps';
```

Comment:
please, use uppercase for importing fragments

Reply:
Updated import name to `InventoryInstance`